File: WinPaletter/GUI/Elements/Others/StripRenderer.vb
Imports System.ComponentModel

Namespace UI.WP

    <Description("StripRenderer fixed to respect dark/light mode")>
    Public Class StripRenderer : Inherits ToolStripSystemRenderer
        Public Sub New() : End Sub
        Protected Overrides Sub OnRenderToolStripBorder(e As ToolStripRenderEventArgs) : End Sub
    End Class

End Namespace

File: WinPaletter/GUI/Elements/Others/StripRenderer.cs
using System.ComponentModel;
using System.Windows.Forms;

namespace WinPaletter.UI.WP
{

    [Description("StripRenderer fixed to respect dark/light mode")]
    public class StripRenderer : ToolStripSystemRenderer
    {
        public StripRenderer()
        {
        }
        protected override void OnRenderToolStripBorder(ToolStripRenderEventArgs e)
        {
        }
    }

}